plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/link_flapping_analyzer.py

```diff
@@ -142,5 +142,7 @@ def plot_link_flapping_last_week(self):
         )
 
     def full_analysis(self):
-        self.get_link_flapping_last_week()
-        return super().full_analysis()
+        super().full_analysis()
```

Comment:
Same thing. Use the infrastructure in the base class to run this.

Reply:
fixed